supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -1,3 +1,35 @@
+[discrete]
+[[AArch64]]
+==== image:images/yes.png[yes] AArch64 (noun)
+
+*Description*: 64-bit version of the ARM architecture. Use this term format when referring to OS and server instances. Including: Linux versions, RHEL, or CoreOS. Use uppercase format in general sentences. 
+
+Example:
+
+_If you are using RHEL with an AArch64 system, run the following command_
```

Comment:
_If you are using RHEL with an AArch64 system, run the following command_

Consider following the style adopted throughout the SSG. Example: 

> For example, "When running the JBoss EAP server in a managed domain".